frontend/src/components/Footer/footer.module.scss

```diff
@@ -207,16 +212,26 @@
   color: #dd2a7b;
 }
 
-.fa-envelope:hover,
-.fa-envelope-own:hover {
-  color: #c71610;
+.fa-instagram:hover,
```

Comment:
any specific reason of modfying the css? Why do we need to change these hover effects?

Reply:
Its a favicon icon, so the icons u see on the footer for instagram is fa-instagram